rust/src/core/polars/mod.rs

```diff
@@ -202,6 +202,24 @@ impl PrivacyNamespaceHelper for Expr {
 
 pub struct DPNamespace(Expr);
 impl DPNamespace {
+    /// Add noise to the expression.
+    ///
+    /// `scale` must not be negative or inf.
+    /// Scale and distribution may be left None, to be filled later by [`make_private_expr`] or [`make_private_lazyframe`].
+    ///
+    /// # Arguments
+    /// * `scale` - Noise scale parameter for the Laplace distribution. `scale` == standard_deviation / sqrt(2).
+    /// * `distribution` - If specified, determines the noise distribution
```

Comment:
What does `scale` mean if it's not Laplace? What is the distribution if it's not provided?

Reply:
I'm not seeing the big picture, but it feels odd that `noise`/`laplace`/`gaussian` is duplicated at the Rust and Python levels... If `laplace`/`gaussian` are just thin wrappers around `noise` in Python, maybe we just need `noise` in Rust? Or maybe there shouldn't be a `noise` in Python, and instead it's handled on the Rust side?

This might be totally off base.

Reply:
Plumbing those thin DSL helpers up from Rust to Python will be a rather technical project compared to just duplicating them. I'm otherwise aiming for API parity. An argument could be made that we *only* need `noise`.